Tests/LibVideo/TestVP9Decode.cpp

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2022, Gregory Bertilson <zaggy1024@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibTest/TestCase.h>
+
+#include <LibVideo/MatroskaReader.h>
+#include <LibVideo/VP9/Decoder.h>
+
+TEST_CASE(webm_in_vp9)
+{
+    auto matroska_document = Video::MatroskaReader::MatroskaReader::parse_matroska_from_file("/usr/Tests/LibVideo/vp9_in_webm.webm"sv);
```

Comment:
This should be able to run in Lagom as well as on serenity, so in the future we should add the "relative file path" hax that other tests that read test inputs do.

Reply:
Is this a relatively simple fix? I could modify this test case if there's a good example to reference. I didn't see anything of the sort when I was looking through other test files. :(

Reply:
Since you've installed the files into `/usr/Tests/LibVideo` (`install(FILES vp9_in_webm.webm DESTINATION usr/Tests/LibVideo)`) they will be available at "./vp9_in_webm.webm". Userland/Utilities/run-tests.cpp will chdir to the parent directory of your test before running it. We can use the same strategy in a future addition to Lagom/CMakeLists.txt to use a relative path there. Some other test files use a "try the abspath then try ./" approach, which could probably use some cleaning up.

So the action here is either to make it "./vp9_in_webm.webm",  or keep that in mind for future on-linux testing approaches.

Reply:
I've made the change to the relative path. Unfortunately I wasn't able to confirm on my machine (using WSL2) that the Lagom tests run since it was encountering a compile error in Zlib.cpp when I run `./Meta/serenity.sh test lagom`:

```/home/zaggy/serenity/Userland/Libraries/LibCompress/Zlib.cpp: In member function ‘void Compress::ZlibCompressor::write_header(Compress::ZlibCompressionMethod, Compress::ZlibCompressionLevel)’:
/home/zaggy/serenity/Userland/Libraries/LibCompress/Zlib.cpp:94:31: error: cannot convert ‘Compress::ZlibCompressionMethod’ to ‘unsigned char:4’ in initialization
   94 |         .compression_method = compression_method,
      |                               ^~~~~~~~~~~~~~~~~~
      |                               |
      |                               Compress::ZlibCompressionMethod
```

I'm unsure yet whether this issue is still present in the latest revision, since this branch is quite outdated now.